service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfChangeControllerTest.java

```diff
@@ -67,129 +56,256 @@
+        final String caseName = "Test";
+        final String solicitorEmail = "solicitor@email.com";
+        final String notificationReference = "localhost/" + CASE_ID;
+        final Respondent otherRespondent = Respondent.builder()
+            .legalRepresentation("Yes")
             .solicitor(RespondentSolicitor.builder()
-                .firstName("Tim")
-                .lastName("Brown")
-                .email("tim.brown@test.com")
-                .organisation(organisation("OLD_ORG"))
-                .build())
-            .build());
-
-        final Element<Respondent> respondent2 = element(Respondent.builder()
-            .party(RespondentParty.builder()
-                .firstName("Second")
-                .lastName("Respondent")
-                .build())
-            .build());
-
-        final Element<Respondent> respondent3 = element(Respondent.builder()
-            .party(RespondentParty.builder()
-                .firstName("Third")
-                .lastName("Respondent")
-                .build())
-            .build());
-
-        final ChangeOrganisationRequest changeRequest = ChangeOrganisationRequest.builder()
-            .organisationToAdd(NEW_ORGANISATION)
-            .caseRoleId(caseRoleDynamicList("[SOLICITORA]"))
+                .firstName("Other")
+                .lastName("Solicitor")
+                .email(solicitorEmail)
+                .organisation(Organisation.builder().organisationID("123").build()).build())
             .build();
 
-        final CaseData caseData = CaseData.builder()
+        final CaseData caseDataBefore = CaseData.builder()
+            .id(CASE_ID)
+            .caseName(caseName)
+            .respondents1(wrapElements(
+                otherRespondent,
+                Respondent.builder()
+                .legalRepresentation("Yes")
+                .solicitor(RespondentSolicitor.builder()
+                    .firstName("Old")
+                    .lastName("Solicitor")
+                    .email(solicitorEmail)
+                    .organisation(Organisation.builder().organisationID("123").build()).build())
+                .build()
+            )).build();
+
+        private final CaseData caseData = CaseData.builder()
             .id(CASE_ID)
-            .changeOrganisationRequestField(changeRequest)
-            .respondents1(List.of(respondent1, respondent2, respondent3))
+            .caseName(caseName)
+            .respondents1(wrapElements(
+                otherRespondent,
+                Respondent.builder()
+                .legalRepresentation("Yes")
+                .solicitor(RespondentSolicitor.builder()
+                    .firstName("New")
+                    .lastName("Solicitor")
+                    .email(solicitorEmail)
+                    .organisation(Organisation.builder().organisationID("123").build()).build())
+                .build()
+            )).build();
+
+        CallbackRequest callbackRequest = CallbackRequest.builder()
```

Comment:
that is ok, but there is method in abstract parent

CallbackRequest callbackRequest = toCallBackRequest(caseData, caseDataBefore);